aten/src/ATen/cudnn/cudnn-wrapper.h

```diff
@@ -6,7 +6,8 @@
 #define STRING(x) STRINGIFY(x)
 
 #if CUDNN_MAJOR < 6
-#pragma message ("CuDNN v" STRING(CUDNN_MAJOR) " found, but need at least CuDNN v6. You can get the latest version of CuDNN from https://developer.nvidia.com/cudnn or disable CuDNN with USE_CUDNN=0")
+#pragma message("CuDNN v" STRING( \
+    CUDNN_MAJOR) " found, but need at least CuDNN v6. You can get the latest version of CuDNN from https://developer.nvidia.com/cudnn or disable CuDNN with USE_CUDNN=0")
```

Comment:
Unrelated but while we're here we might want to update this to 8 or major >= 8 and minor >= 5 as older versions have not really been supported for a while now